.github/workflows/build.yml

```
@@ -87,11 +80,103 @@ jobs:
           path: |
             node_modules
             */*/node_modules
+            ${{ github.workspace }}/packages/**/build
+            ${{ github.workspace }}/packages/**/dist
+            ${{ github.workspace }}/packages/**/esm
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
       - run: yarn install
-      - uses: actions/download-artifact@v2
-        with:
-          name: build
-          path: ${{ github.workspace }}/packages
       - name: Unit Tests
-        run: yarn test
+        run: yarn test --ignore="@sentry/ember"
+      - uses: codecov/codecov-action@v1
+
+  job_browserstack_test:
+    name: Browserstack
+    needs: job_build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
```

Maybe to be more explicit?:
```suggestion
            ${{ github.workspace }}/node_modules
            ${{ github.workspace }}/packages/*/node_modules
```